<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="KAD-I-032" ignored="0" name="Default test" tests="2" failures="1" timestamp="01 Nov 2018 10:36:09 GMT" time="36.106" errors="0">
  <testcase name="Login" time="12.105" classname="maintest2.Basic"/>
  <testcase name="dataEntry" time="18.866" classname="maintest2.Basic">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Report Generation Pending] but found [After Verification Initiation],
	expected [Report Generation Pending] but found [After Verification Initiation],
	expected [Report Generation Pending] but found [After Verification Initiation],
	expected [Report Generation Pending] but found [After Verification Initiation],
	expected [Report Generation Pending] but found [Verification Pending],
	expected [Report Generation Pending] but found [After Verification Initiation],
	expected [Report Generation Pending] but found [After Verification Initiation],
	expected [Report Generation Pending] but found [After Verification Initiation],
	expected [Report Generation Pending] but found [After Verification Initiation],
	expected [Report Generation Pending] but found [After Verification Initiation],
	expected [Report Generation Pending] but found [After Verification Initiation],
	expected [Report Generation Pending] but found [After Verification Initiation],
	expected [Report Generation Pending] but found [Verification Pending]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Report Generation Pending] but found [After Verification Initiation],
expected [Report Generation Pending] but found [After Verification Initiation],
expected [Report Generation Pending] but found [After Verification Initiation],
expected [Report Generation Pending] but found [After Verification Initiation],
expected [Report Generation Pending] but found [Verification Pending],
expected [Report Generation Pending] but found [After Verification Initiation],
expected [Report Generation Pending] but found [After Verification Initiation],
expected [Report Generation Pending] but found [After Verification Initiation],
expected [Report Generation Pending] but found [After Verification Initiation],
expected [Report Generation Pending] but found [After Verification Initiation],
expected [Report Generation Pending] but found [After Verification Initiation],
expected [Report Generation Pending] but found [After Verification Initiation],
expected [Report Generation Pending] but found [Verification Pending]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at maintest2.Basic.dataEntry(Basic.java:108)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- dataEntry -->
</testsuite> <!-- Default test -->
